### ISSUE TITLE
Fix some issues with Internet Explorer

### ISSUE THREAD

**Issue by Gottox** [Gottox/terminal.js#64]
Currently, a few tests fail with Internet Explorer 8. It seems this isn't much work to fix.

For 0.2.x, support on IE would be nice.

**Comment by abarik1981** [comment]
Yes I agree, I see that the frontend CLI doesn't work on IE9: https://github.com/Gottox/node-webterm/issues/5
